src/modules/facility-investor/facility-investor.service.ts

```diff
@@ -38,4 +40,24 @@ export class FacilityInvestorService {
     };
     await this.acbsFacilityPartyService.createPartyForFacility(facilityIdentifier, facilityPartyToCreateInAcbs, idToken);
   }
+
+  async getInvestorsForFacility(facilityIdentifier: UkefId): Promise<GetFacilityInvestorsResponse> {
+    const idToken = await this.acbsAuthenticationService.getIdToken();
+    const portfolio = PROPERTIES.GLOBAL.portfolioIdentifier;
```

Comment:
can we use for this something like:
```
const { portfolioIdentifier } = PROPERTIES.GLOBAL;
```
this so that we don't introduce a new variable

Reply:
Changed in move and couple other places, but old code style is sued in more places, we can replace them later.